queries/highlights.scm

```diff
@@ -80,6 +94,9 @@
 
 (lifetime (identifier) @label)
 
+"as" @keyword
```

Comment:
There is a line below saying `"as" @operator`. Is it appropriate to have both, or does one need to be removed?

Reply:
Oh yeah, there probably doesn't need to be both. As for which color we want, I'm not sure. I can't remember how the two things are colored on GitHub.com. Maybe `keyword` is more appropriate though?

Reply:
```rust
fn main() {
    let _ = self as &dyn T;
}
```

![Screenshot from 2022-07-18 16-16-52](https://user-images.githubusercontent.com/1940490/179632524-424b098e-2120-4b62-bbac-7c27e640bae3.png)

In the default light theme, keyword is `#cf222e` while operator is `#0550ae`.

Here is what it would look like as keyword. Seems okay either way.

![Screenshot from 2022-07-18 16-22-09](https://user-images.githubusercontent.com/1940490/179633035-ba12ca24-0aba-4807-8e91-2d7f5386e8bb.png)

Reply:
Sorry, I didn't see the existing rule.

`as` has three completely unrelated uses in Rust.

- type cast: `nvalues as u32`
- import alias: `use std::collections::HashMap as Map;`
- qualify a method with both type and trait: `<TcpStream as Write>::write_all(s, bytes)?;`

In all cases `as` appears between two things, so `operator` isn't bad. I think `keyword` is a little better though. I'll fix it that way.

(It would also be possible to use `operator` only for casts, but that seems excessive to me.)

Reply:
Removed the `@operator` rule.